database/mongodb.go

```diff
@@ -57,6 +60,12 @@ func (db *MongoDB) init() (err error) {
 }
 
 func (db *MongoDB) build() string {
+	if len(db.uri) > 0 {
+		return mongodumpCli + " " +
+			"--uri=" + db.uri + " " +
```

Comment:
If the `uri` is not present, there will error, so you must check and only if `uri` present, then add `--uri` option.

Reply:
Oh, you have done that.